app/controllers/stories_controller.rb

```diff
@@ -131,6 +136,8 @@ def handle_base_index
     assign_podcasts
     assign_classified_listings
     @article_index = true
+    @featured_story = (@featured_story || Article.new)&.decorate
+    @stories = @stories&.decorate
```

Comment:
Are we decorating here because we plan on adding properties to just the stories in the feed? Is that work coming in another PR?

Reply:
I'm hoisting up the decorate calls on these objects that previously was done at app/views/articles/index.html.erb:87 & 88. the `.decorate` method add methods from the `ArticleDecorator` class to these `Article` objects, and it seems odd to me the controller wouldn't just delivery them to the view in that state versus making it the view's responsibility.